packages/appstore/src/components/wallet-transfer/transfer-account-selector/transfer-account-selector.tsx

```diff
@@ -96,20 +100,12 @@ const TransferAccountSelector = ({
     const [is_list_modal_open, setIsListModalOpen] = React.useState(false);
     const [selected_account, setSelectedAccount] = React.useState<TTransferAccount | undefined>(value);
 
-    React.useEffect(() => {
```

Comment:
remove this useEffect because now we pass a key to TransferAccountSelector which will render the proper ui